docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,35 +1,372 @@
+
+- **Start from a template** - Use one of our quickstart templates to get up and running. All templates come with CI/CD already configured and will be cloned to a new GitHub repository.
+
+- **Import an existing project** - Import an existing Dagster project using our GitHub app or the [dagster-cloud CLI](/dagster-cloud/developing-testing/dagster-cloud-cli). **Note**: If using the GitHub app, Dagster Cloud will automatically set up CI/CD for you.
+
+<TabGroup>
+<TabItem name="Use a template">
+
+#### Using a template
+
+1. Click **Select** to select a template.
+2. Sign in to your GitHub account, if prompted.
+3. In Dagster Cloud, define the following:
+   - **Git scope** - Select the organization or account to create the new repository in.
+   - **Repository Name** - Enter a name for the new repository.
+   - Check the **Make git repository private** box to make the repository private.
+4. When finished, click **Clone and deploy**.
+
+When finished, [continue to the next step](#step-4-set-up-environment-variables-and-secrets).
+
+</TabItem>
+<TabItem name="Import an existing project">
+
+#### Import an existing project
+
+To import an existing project, you can use Dagster's GitHub app or the dagster-cloud CLI.
+
+<TabGroup>
+<TabItem name="GitHub app">
+
+**Using the GitHub app**
+
+Using the GitHub app to import an existing project also sets up CI/CD for you.
+
+Before you get started, note that your project must have a `dagster_cloud.yaml` file in order for Dagster Cloud to deploy it. If your project doesn't contain this file, you'll need to add it before proceeding.
+
+In the root of your project, create a `dagster_cloud.yaml` with the following code:
+
+```yaml
+locations:
+  - location_name: my_location_name # the name of the code location
+    code_source:
+      package_name: hackernews # the name of the python package associated with the code location
+```
+
+After you've committed the file to the repository, come back to Dagster Cloud to complete the import process:
+
+1. Click the **Import an existing project** tab.
+2. Sign in to your GitHub account, if prompted.
+3. In Dagster Cloud, define the following:
+   - **Git scope** - Select the organization or account that the repository is in.
+   - **Repository** - Select the repository.
+4. Click **Deploy**.
+
+When finished, [continue to the next step](#step-4-set-up-environment-variables-and-secrets).
+
+</TabItem>
+<TabItem name="dagster-cloud CLI">
+
+**Using the dagster-cloud CLI**
+
+<Note>
+  Using the dagster-cloud CLI requires a recent version of Python 3 and Docker.
+</Note>
+
+To complete this step using the CLI, you can use your own Dagster code or the [Dagster Cloud Serverless Quickstart](https://github.com/dagster-io/dagster-cloud-serverless-quickstart). The quickstart is a template with everything you need to get started using Serverless in Dagster Cloud, including CI/CD configuration and the required `dagster_cloud.yaml` file.
```

Comment:
lets use this quickstart instead: https://github.com/dagster-io/quickstart-etl

Reply:
Updated